### PR TITLE
Hypnotemp Plushies

### PR DESCRIPTION
**Pull request by Cprn-Sprkl**
# Description

Adds seven new HypnoTemp Staffer absorbent plushies. They can each hold 30u of liquid and come prefilled with different solutions for each variant. These plushies have been made findable in maintenance closets, maintenance wall lockers, and inside of prize balls.

The solutions have mostly mild and fun benefits such as small amounts of healing, slight speed boosts for a few seconds, and laughter. They're made up of mainly readily available chemicals with the exception of omnizine, which is available in the CMO variant plushie. The amount should be comparable to round start herbals while having no deterministic way of acquiring the CMO variant plushie.

<details><summary><h1>Media</h1></summary>
<p>

HTPPreview

</p>
</details>

---

# Changelog

:cl: sprkl
- add: Seven variants of HypnoTemp Staffer plushies can now be found in station lockers and prize balls.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml [hunk 1]
```diff
@@ -288,6 +288,15 @@
...
         orGroup: Prize
+      - id: PlushieHypnoTempLO
+        prob: 0.80
+        orGroup: Prize
+      - id: PlushieHypnoTempHoP
+        prob: 0.80
+        orGroup: Prize
+      - id: PlushieHypnoTempCMO
+        prob: 0.80
+        orGroup: Prize
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml:299]:
```suggestion
      # HypnoTemp Merch - Start
      - id: PlushieHypnoTempLO
        prob: 0.80
        orGroup: Prize
      - id: PlushieHypnoTempHoP
        prob: 0.80
        orGroup: Prize
      - id: PlushieHypnoTempCMO
        prob: 0.80
        orGroup: Prize
      #HypnoTemp Merch - End
```

File: Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml [hunk 2]
```diff
@@ -367,6 +376,12 @@
       - id: MrDips
         prob: 0.50
         orGroup: Prize
+      - id: PlushieHypnoTempMysta
+        prob: 0.50
+        orGroup: Prize
+      - id: PlushieHypnoTempHoS
+        prob: 0.50
+        orGroup: Prize
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml:384]:
```suggestion
      # HypnoTemp Merch - Start
      - id: PlushieHypnoTempMysta
        prob: 0.50
        orGroup: Prize
      - id: PlushieHypnoTempHoS
        prob: 0.50
        orGroup: Prize
       #HypnoTemp Merch - End
```

File: Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml [hunk 3]
```diff
@@ -398,3 +413,6 @@
       - id: BwoinkHammer
         prob: 0.30
         orGroup: Prize
+      - id: PlushieHypnoTempCap
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml:416]:
```suggestion
      - id: PlushieHypnoTempCap # Floofstation
```

File: Resources/Textures/Floof/Objects/HypnoTempPlushies/staffer_plushies.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": 1,
+    "license": "CC BY-NC-SA 4.0",
+    "copyright": "JTCircus",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Objects/HypnoTempPlushies/staffer_plushies.rsi/meta.json:4]: I suggest providing more information, like discord ID, github account, ect... this is fine too

File: Resources/Prototypes/Floof/Entities/Objects/HypnoTempPlushies/HypnoTempPlushies.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Objects/HypnoTempPlushies/HypnoTempPlushies.yml:1]: Pretty sure we will not allo Hypo Plush, but we will review this after some talk.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Some minors changes code wise.
While this is a self-insert, we will talk with the team regarding those.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Tests fails are valid.

**Comment by Fansana** [comment]
can you add this to the plush vendor?

**Review by Fansana — APPROVED** [review]

**Comment by FoxxoTrystan** [comment]
While they are in the random prizeball ticket, they are not in the prizevendor.

File: Resources/Prototypes/Floof/Entities/Objects/HypnoTempPlushies/HypnoTempPlushies.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Objects/HypnoTempPlushies/HypnoTempPlushies.yml:1]: Actully im against this, Hypno Plushies for everyone that look in maintenance is a big nono.

Il prefer the plushie is not a hypno item or is not findable in maintenance.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Floof/Entities/Objects/HypnoTempPlushies/HypnoTempPlushies.yml
Comment (FoxxoTrystan) [reply Resources/Prototypes/Floof/Entities/Objects/HypnoTempPlushies/HypnoTempPlushies.yml:1]: They act like a hypo item correct? they can contain solutions.

**Review by FoxxoTrystan — APPROVED** [review]

**Pull request merged**